docs/Installation.md

```diff
@@ -39,22 +39,31 @@ The `Project` subdirectory contains many [example environments](Learning-Environ
 to help you get started.
 
 ### Package Installation
+ML-Agents C# SDK is transitioning to a Unity Package.  While we are working on getting into the
+official packages list, you can add the `com.unity.ml-agents` package to your project by
+navigating to the menu `Window`  -> `Package Manager`.  In the package manager window click
+on the `+` button.
 
-If you intend to copy the `com.unity.ml-agents` folder in to your project, ensure that
-you have the [Barracuda preview package](https://docs.unity3d.com/Packages/com.unity.barracuda@0.3/manual/index.html) installed.
+<p align="center">
+  <img src="images/unity_package_manager_window.png"
+       alt="Linux Build Support"
+       width="500" border="10" />
+</p>
+**NOTE:** In Unity 2018.4 it's on the bottom left and in Unity 2019.3 it's on the top left.
```

Comment:
Weird, this shows up as literal `**`'s if you view the file. Maybe the `:` is confusing it?

Reply:
looks like it needed an extra line between the `</p>` and the `**`